app/include/dt-bindings/zmk/modifiers.h

```diff
@@ -25,4 +25,8 @@
 #define RC(keycode) APPLY_MODS(MOD_RCTL, keycode)
 #define RS(keycode) APPLY_MODS(MOD_RSFT, keycode)
 #define RA(keycode) APPLY_MODS(MOD_RALT, keycode)
-#define RG(keycode) APPLY_MODS(MOD_RGUI, keycode)
\ No newline at end of file
+#define RG(keycode) APPLY_MODS(MOD_RGUI, keycode)
+
+#define IS_MOD(usage_page, keycode)                                                                \
+    (usage_page == HID_USAGE_KEY && keycode >= HID_USAGE_KEY_KEYBOARD_LEFTCONTROL &&               \
+     keycode <= HID_USAGE_KEY_KEYBOARD_RIGHT_GUI)
```

Comment:
Should this be a macro or a function?  @joelspadin, any thoughts please?

Reply:
I don't see anything in here that would require it to be a macro, so my vote is function. I can almost guarantee you that the compiler will inline this if you make it a function, and the resulting binary will be very close to if not identical to with a macro.

Reply:
would you define that function in the header file, and if not, where should we put it? I would like to keep this close together with the `SELECT_MODS` macro and friends.

Reply:
Two ways you can do this:

1. Define it in the header as an inline function. (Needs to be inline so the linker won't complain when it sees that every C file that includes modifiers.h has its own copy of the function.)
2. Declare it in the header and define it in some other C file.

Reply:
So, the challenge here, is this header here *really* should only be macros, and those for keymaps. Either way this code shouldn't really be here. Perhaps inline function in `app/include/zmk/keys.h`?